### ISSUE TITLE
[BUG] setup.py limits dependencies to old versions

### ISSUE THREAD

**Issue by arjunvenkatraman** [hackergram/xetrapal#8]
**Describe the bug**
When I install using setup.py, the script first installs dependencies using pip and then tries to
 install them again using easy_install. While this does verify those dependencies that are already installed it also sets them to "installed by easy_install" which limits them to older versions
**To Reproduce**
Run ``` sudo -H python setup.py install```

**Expected behavior**
Only pip should be used to install the dependencies

**Screenshots**
Using /usr/local/lib/python2.7/dist-packages
Searching for cachetools==3.0.0
Best match: cachetools 3.0.0
Adding cachetools 3.0.0 to easy-install.pth file

Using /usr/local/lib/python2.7/dist-packages
Finished processing dependencies for xetrapal==0.0

**rasa-core 0.12.3 has requirement coloredlogs~=10.0, but you'll have coloredlogs 9.0 which is incompatible.
rasa-core 0.12.3 has requirement python-telegram-bot~=10.0, but you'll have python-telegram-bot 7.0.1 which is incompatible.
rasa-core-sdk 0.12.1 has requirement coloredlogs~=10.0, but you'll have coloredlogs 9.0 which is incompatible.
pyinquirer 1.0.3 has requirement Pygments>=2.2.0, but you'll have pygments 2.1 which is incompatible.**

Collecting python_telegram_bot==7.0.1 (from -r requirements.txt (line 1))

**Desktop (please complete the following information):**
 - OS: Ubuntu 18.04
 

**Smartphone (please complete the following information):**
NA

**Additional context**
NA

**Comment by arjunvenkatraman** [comment]
Has this been fixed? I don't see a commit that references this

**Comment by karan100010** [comment]
Yes, this is fixed will make sure all bugs are mentioned in there corresponding commits. Apologies for the confusion.
